lib/chef/knife/vra_server_create.rb

```diff
@@ -22,20 +22,20 @@
 require "chef/knife/cloud/server/create_command"
 require "chef/knife/cloud/server/create_options"
 require_relative "cloud/vra_service_options"
+require_relative "cloud/vra_service_helpers"
 
 class Chef
   class Knife
     class Cloud
       class VraServerCreate < ServerCreateCommand
-        include VraServiceHelpers
         include VraServiceOptions
         include ServerCreateOptions
+        include VraServiceHelpers
```

Comment:
If you can make it work by putting the require and the include in the deps block you won't take the penalty of loading that code every time you run any old knife command.

Reply:
That was my first attempt but it blew up several tests that call `validate_params!` directly which does not trigger the `deps` block.